xrdsst/controllers/tls.py

```diff
@@ -0,0 +1,234 @@
+from datetime import datetime
+import os
+import cement.utils.fs
+from cement import ex
+from xrdsst.api import SystemApi
+from xrdsst.controllers.base import BaseController
+from xrdsst.core.util import parse_argument_list
+from xrdsst.api_client.api_client import ApiClient
+from xrdsst.resources.texts import texts
+from xrdsst.rest.rest import ApiException
+from xrdsst.models.distinguished_name import DistinguishedName
+
+
+class DownloadedTLSListMapper:
+    @staticmethod
+    def headers():
+        return ['SECURITY SERVER', 'LOCATION']
+
+    @staticmethod
+    def as_list(dwn_tls):
+        return [dwn_tls.security_server, dwn_tls.fs_loc]
+
+    @staticmethod
+    def as_object(dwn_tls):
+        return {
+            'security_server': dwn_tls.security_server,
+            'fs_loc': dwn_tls.fs_loc
+        }
+
+
+class DownloadedTLS:
+    def __init__(self, security_server, fs_loc):
+        self.security_server = security_server
+        self.fs_loc = fs_loc
+
+
+class TlsController(BaseController):
```

Comment:
Please consider changing the name command "xrdsst tls" to "xrdsst internal-tls" as TLS is too general. Hence, also TlsController should become InternalTlsController.